src/app/containers/Timestamp/index.test.jsx

```diff
@@ -111,4 +104,118 @@ describe('Timestamp', () => {
       /Updated [0-9]{1,2} \w+ [0-9]{4}/,
     );
   });
+
+  describe('time dependent tests', () => {
+    // eslint-disable-next-line no-unused-vars
+    let clock;
+    describe('tests after 10 am', () => {
+      const inBritishSummerTime = isBritishSummerTime(Date.now());
```

Comment:
Rather than using `lolex` to manipulate time, can we do this with Jest mocks?

Look at how I've done it here: https://github.com/bbc/simorgh/pull/1505/files#diff-aec09657c840fdae03a6bf73cec50c31R37

Should be able to do something like:

```js
const originalDateNow = Date.now;
Date.now = jest.fn(() => 1496235600000);

originalDateNow(); // if you need the _actual_ Date.now() value
Date.now(); // if you want the mocked value of 1496235600000
```